### DISCUSSION TITLE
Log custom events for Partner Center

### DISCUSSION THREAD

**Discussion by dhq-boiler** [dotnet/wpf#5971] [Q&A]
I have a question.
I'm developing an application in WPF for .NET 6 and just recently submitted it to the Microsoft store to publish the app.
https://github.com/dhq-boiler/boiler-s-Graphics

https://docs.microsoft.com/en-us/windows/uwp/monetize/log-custom-events-for-dev-center
After reading this article, I knew that I could log custom events in Partner Center, but when I tried to implement it, it didn't work.

Is it impossible in WPF in the first place? Please let me know if you have any good solution.

**Comment by vishalmsft** [comment]
Hi @dhq-boiler . This question is more about Desktop application vs UWP application and ability to use those APIs. Desktop application do not have direct access to UWP APIs. You may want to use soft of bridge or service to log the events.

Moving this to discussion.
